app/src/main/res/layout/activity_main.xml

```diff
@@ -15,5 +15,34 @@
             app:layout_constraintLeft_toLeftOf="parent"
             app:layout_constraintRight_toRightOf="parent"
             app:layout_constraintTop_toTopOf="parent" android:id="@+id/textView"/>
+    <Button
```

Comment:
Please use default code style for xml and align this file once again

![image](https://user-images.githubusercontent.com/16294951/54492627-4ee14500-48d9-11e9-9455-1b4962d68625.png)

Reply:
@egorikftp done